dev/reconstruct_source.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/python
+#
+# Copyright 2017 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import sys
+import yaml
+
+from spinnaker.run import check_run_quick
+
+COMPONENTS = [
+  'clouddriver',
+  'deck',
+  'echo',
+  'front50',
+  'gate',
+  'igor',
+  'orca',
+  'rosco',
+  'fiat',
+  'spinnaker-monitoring',
+  'spinnaker'
```

Comment:
spinnaker does have a BOM entry, but we don't build a package from the source. It's just to track the tools we used to build it.